src/sentry/static/sentry/app/api.tsx

```diff
@@ -62,8 +61,10 @@ const ALLOWED_ANON_PAGES = [
   /^\/join-request\//,
 ];
 
-export function initApiClientErrorHandling() {
-  jQuery(document).ajaxError(function (_evt, jqXHR) {
+const globalErrorHandlers: ((jqXHR: JQueryXHR) => void)[] = [];
+
+export const initApiClientErrorHandling = () =>
```

Comment:
We only want to setup this error handling for the app. I think this global approach is probably OK. But am open to thoughts.

Reply:
@EvanPurkhiser confused about why we need this if we just have one global error handler.

Reply:
We don't turn this on in sentry admin